data/strings/strings.txt

```diff
@@ -868,24 +868,276 @@
+    en = Created by enthusiasts and volunteers with the power of open-source.
+    ar = . ﺔﺣﻮﺘﻔﻤﻟﺍ ﺭﺩﺎﺼﻤﻟﺍ ﻊﻤﺘﺠﻣ ﻦﻣ ﺓﺍﻮﻫ ﺔﻄﺳﺍﻮﺑ ﺎﻫﺅﺎﺸﻧﺇ ﻢﺗ
+    be = Створана энтузіястамі з супольнасці адкрытых праграм.
+    bg = Създаден от ентусиасти от общността с отворен код.
+    ca = Creat per entusiastes de la comunitat de codi obert.
+    cs = Vytvořeno nadšenci z open-source komunity.
+    da = Skabt af entusiaster fra open source-fællesskabet.
+    de = Erstellt von Enthusiasten aus der Open-Source-Community
+    el = Δημιουργήθηκε από λάτρεις της κοινότητας ανοιχτού κώδικα.
+    es = Creado por entusiastas de la comunidad de código abierto.
+    et = Loodud avatud lähtekoodiga kogukonna entusiastide poolt.
+    eu = Kode irekiko komunitateko zaleek sortua.
+    fa = .ﺯﺎﺑ ﻊﺒﻨﻣ ﻪﻌﻣﺎﺟ ﺯﺍ ﻥﺍﺪﻨﻣ ﻪﻗﻼﻋ ﻂﺳﻮﺗ ﻩﺪﺷ ﺩﺎﺠﯾﺍ
+    fi = Avoimen lähdekoodin yhteisön harrastajien luoma.
+    fr = Créé par des passionnés de la communauté open-source
+    he = .חותפה דוקה תליהקמ םיבבוח ידי לע רצונ
+    hu = A nyílt forráskódú közösség rajongói készítettek.
+    id = Dibuat oleh penggemar dari komunitas sumber terbuka.
+    it = Creato da appassionati della comunità open-source.
+    ja = オープンソース コミュニティの愛好家によって作成されました。
+    ko = 오픈 소스 커뮤니티의 매니아들이 만든.
+    mr = मुक्त-स्रोत समुदायातील उत्साही लोककां. तयार केलेले
+    nb = Laget av entusiaster fra åpen kildekode-fellesskapet.
+    nl = Gemaakt door enthousiastelingen uit de open-sourcecommunity.
+    pl = Stworzony przez entuzjastów ze społeczności open source.
+    pt = Criado por entusiastas da comunidade de código aberto.
+    pt-BR = Criado por entusiastas da comunidade de código aberto.
+    ro = Creat de entuziaști din comunitatea open-source.
+    ru = Создано энтузиастами из сообщества open-source.
+    sk = Vytvorené nadšencami z open-source komunity.
+    sv = Skapad av entusiaster från öppen källkodsgemenskap.
+    sw = Imeundwa na wakereketwa kutoka kwa jumuiya ya chanzo huria.
+    th = สร้างโดยผู้ที่ชื่นชอบจากชุมชนโอเพ่นซอร์ส.
+    tr = Açık kaynak topluluğunun meraklıları tarafından oluşturuldu.
+    uk = Створено ентузіастами спільноти відкритих програм.
+    vi = Được tạo bởi những người đam mê từ cộng đồng mã nguồn mở.
+    zh-Hans = 由开源社区的爱好者创建.
+    zh-Hant = 由開源社區的愛好者創建.
+
+  [about_report_to_osm]
+    tags = android
+    comment = Text in About screen
+    en = Please fix all map issues at OpenStreetMap.org.
+    ar = .OpenStreetMap.org ﻊﻗﻮﻣ ﻰﻠﻋ ﻂﺋﺍﺮﺨﻟﺍ ﺕﺎﻧﺎﻴﺑ ﺕﻼﻜﺸﻣ ﺢﻠﺻﺃ
+    be = Вырашыце праблемы з дадзенымі карты на OpenStreetMap.org.
+    bg = Коригирайте проблеми с картографските данни на OpenStreetMap.org.
+    ca = Solucioneu problemes de dades de mapes a OpenStreetMap.org.
+    cs = Opravte problémy s mapovými daty na OpenStreetMap.org.
+    da = Løs problemer med kortdata på OpenStreetMap.org.
+    de = Kartendatenprobleme auf OpenStreetMap.org beheben.
+    el = Διορθώστε προβλήματα δεδομένων χάρτη στο OpenStreetMap.org.
+    es = Solucionar problemas de datos de mapas en OpenStreetMap.org.
```

Comment:
This Spanish string can be improved as “Solucione problemas en los datos cartográficos en OpenStreetMap.org.”